tests/unittests/TensorsTest.cpp

```diff
@@ -1293,3 +1293,41 @@ TEST(Tensor, differentAlignment) {
   T2H.at({1, 0}) = 1;
   EXPECT_FALSE(T1.isEqual(T2));
 }
+
+// Check that both write / read of tensors data from / to raw-text files is
+// working properly.
+TEST(Tensor, accessToRawTextFile) {
+  Tensor Tref = {0.75f,  0.23f, 0.76f,  0.99f,  1.00f,
```

Comment:
Glow usually uses camelCase. It would  mean `tRef` in this case.

Reply:
Done.